docs/source/ops/connect/content.rst

```diff
@@ -0,0 +1,27 @@
+.. Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+.. _ops-connect-content_index:
+
+Log aggregation on Kubernetes
+=============================
+
+If configured so, every daml service can output it's logs in JSON format.
+This makes it easy to analyze these with search engines like `ElasticSearch <https://www.elastic.co/elasticsearch/>`_ in combination with `Kibana <https://www.elastic.co/kibana/>`_.
+
+To collect all logs for analyzing them e.g. in your `ElasticSearch` instance, one easy way to achieve this is using `Filebeat <https://www.elastic.co/beats/filebeat>`_.
+It regulary collects the logs of your containers and ingests them into `Elasticsearch`, `Logstash <https://www.elastic.co/logstash/>`_ or `Kafka <https://kafka.apache.org/>`_ etc. and the best is, it doesn't require any additional configuration for the daml services.
+
+You can find external documentation on, how to setup `ElasticSearch` with `Filebeat` and `Kibana` for analyzing logs on your Kubernetes cluster `here <https://www.deepnetwork.com/blog/2020/01/27/ELK-stack-filebeat-k8s-deployment.html>`_.
+
+Daml Metrics options
```

Comment:
Similar here, this should show an example of integrating this with prometheus and not just list CLI flags.

Reply:
Same as for the other, Minikube cluster setup?

Reply:
Side note, I would like to keep these sections and add an additional example section because I think it is good to have an overview + example.

Reply:
Great, many thanks for the input. Then I can proceed further here :)